storybook/.storybook/vscode-theme.css

```diff
@@ -0,0 +1,197 @@
+/* VSCode theme variables for Storybook */
+/* This file provides the VSCode CSS variables that components expect */
+
+:root {
+	/* Editor colors */
+	--vscode-editor-background: #1e1e1e;
+	--vscode-editor-foreground: #d4d4d4;
+	--vscode-sideBar-background: #252526;
+	--vscode-sideBar-foreground: #cccccc;
```

Comment:
These are roughly right. They are defined by the theme so as long as we have a representative light/dark we should be good!